client/src/components/RecyclingInfo.js

```diff
@@ -4,23 +4,18 @@ import { Col, PageHeader } from 'react-bootstrap';
 
 const RecyclingInfo = () => (
   <Col xs={10} xsOffset={1} md={6} mdOffset={0} className="home-section-container">
-    <div>
-      <PageHeader className="home-section-page-header">Wait, But Why?</PageHeader>
-    </div>
-    <div>
-      <h3 className="home-section-title">
-        Denver's recycling rate is only 20%, while the national average is 35%.
-      </h3>
+    <div className="home-section-div">
```

Comment:
** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?